src/test/java/Homework19.java

```diff
@@ -9,5 +9,4 @@ public static void deletePlaylist() throws InterruptedException {
         testPlaylist();
         deleteButton();
         playlistNotExist();
```

Comment:
Try to find other locatro for this method

In this case the locator (div.alertify-logs.top.right) will return an element even if message not displayed
![Dee Yan-Key - Wanderer above the Sea of Fog ♫ Koel - Google Chrome](https://user-images.githubusercontent.com/49252454/211904212-ef3b592f-9556-4a7d-825b-53f9ef1d706b.jpg)